js/app/interfaces/card.js

```diff
@@ -272,6 +272,18 @@ const Card = new Lang.Interface({
         label.visible = !!text;
     },
 
+    set_duration_label: function (label, duration) {
```

Comment:
There is nothing really card specific with this method right? Wouldn't it make more sense as helper in utils.js? Specially since we only use it for one specific card.

What do you think?